src/logger.js

```diff
@@ -154,24 +156,39 @@ export function Logger({ url, prefix, logLevel = DEFAULT_LOG_LEVEL, transport =
             }
 
             if (amplitudeApiKey) {
-                transport({
-                    method:  'POST',
-                    url:     AMPLITUDE_URL,
-                    headers: {
-                        'content-type': 'application/json'
-                    },
-                    json: {
-                        api_key: amplitudeApiKey,
-                        events:  tracking.map((payload : Payload) => {
-                            // $FlowFixMe
-                            return {
-                                event_type:       payload.transition_name || 'event',
-                                event_properties: payload,
-                                ...payload
-                            };
-                        })
-                    }
-                }).catch(noop);
+                const data = {
+                    api_key: amplitudeApiKey,
+                    events:  tracking.map((payload : Payload) => {
+                        // $FlowFixMe
+                        return {
+                            event_type:       payload.transition_name || 'event',
+                            event_properties: payload,
+                            ...payload
+                        };
+                    })
+                };
+                if (window.navigator.sendBeacon(url, data)) {
```

Comment:
Can we get a null check for `window.navigator.sendBeacon`? It's not present in IE

Reply:
We should also respect the `enableSendBeacon` flag. Maybe we can define a `canUseSendBeacon() : boolean` helper

Reply:
Also thinking about it -- does it make sense to encapsulate all of this inside `transport()`, and just pass a `useBlob: false`?